dask/array/tests/test_testing.py

```diff
@@ -0,0 +1,19 @@
+import sys
+
+import pytest
+import numpy as np
+
+import dask.array as da
+from dask.array.utils import assert_eq
+
+
+@pytest.mark.skipif(sys.flags.optimize == 2, reason="Assertions disabled.")
```

Comment:
I think this should just be `sys.flags.optimize`. Assertions are disabled even at level 1 (it just happens that Travis only tests with level 2).